### PR TITLE
Feature delete account

### PR DESCRIPTION
**Pull request by keviny22**
@brettweavnet-intuit could you review please?

### CODE REVIEW

File: lib/aws_account_utils/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module AwsAccountUtils
-  VERSION = "0.1.2"
+  VERSION = "0.1.3"
```
Comment (brettweavnet-intuit) [inline lib/aws_account_utils/version.rb]: @keviny22  we do the version bump as a separate commit on master to make it easily identifiable.

**Comment by brettweavnet-intuit** [comment]
@keviny22 looks good.  Remove the version bump and let me know once it's ready to merge.

**Comment by keviny22** [comment]
@brettweavnet-intuit removed version

**Comment by brettweavnet-intuit** [comment]
@keviny22 merged.  you can now push the latest tag to master.
